lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -656,7 +661,8 @@ def set_payload(record, entry)
       # available, or if the only remainaing key is 'message'.
       if @service_name == CLOUDFUNCTIONS_SERVICE && @cloudfunctions_log_match
         entry['textPayload'] = @cloudfunctions_log_match['text']
-      elsif @service_name == CLOUDFUNCTIONS_SERVICE && record.key?('log')
+      elsif (@service_name == CLOUDFUNCTIONS_SERVICE ||
+             @service_name == CONTAINER_SERVICE) && record.key?('log')
```

Comment:
I actually think the control that doing it this way gives us to manually curate what makes it through is better.

I don't want to start spitting out json payloads at users in the log viewer just because the metadata filter plugin starts adding an additional field to the record that we miss when making a new fluentd container image.